composer.json

```
@@ -58,20 +58,16 @@
         "phpstan/phpstan-phpunit": "^1.0 || ^2.0",
         "phpstan/phpstan-strict-rules": "^1.0 || ^2.0",
         "phpstan/phpstan-symfony": "^1.0 || ^2.0",
-        "phpunit/phpunit": "^10.5.54 || ^11.5.38 || ^12.3.10",
-        "psalm/plugin-phpunit": "^0.18 || ^0.19",
-        "psalm/plugin-symfony": "^5.0",
+        "phpunit/phpunit": "^11.5.38 || ^12.3.10",
         "rector/rector": "^1.1 || ^2.0",
         "sonata-project/block-bundle": "^5.0",
         "sonata-project/entity-audit-bundle": "^1.1",
-        "symfony/browser-kit": "^6.4 || ^7.1",
-        "symfony/css-selector": "^6.4 || ^7.1",
+        "symfony/browser-kit": "^6.4 || ^7.3 || ^8.0",
+        "symfony/css-selector": "^6.4 || ^7.3 || ^8.0",
         "symfony/panther": "^2.1.1",
-        "symfony/templating": "^6.4 || ^7.1",
-        "symfony/uid": "^6.4 || ^7.1",
-        "symfony/yaml": "^6.4 || ^7.1",
-        "vimeo/psalm": "^5.7 || ^6.10",
-        "weirdan/doctrine-psalm-plugin": "^2.0"
+        "symfony/templating": "^6.4 || ^7.3 || ^8.0",
```

```suggestion
        "symfony/templating": "^6.4",
```

there is no v7 or 8 release

---

Should probably be removed, i don't see `templating` used anywhere?

---

yeah need to check :+1: